qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -237,6 +237,11 @@
         - gagent_vss_status:
             only Windows
             gagent_check_type = vss_status
+        - gagent_with_migrate:
+            only virtio_serial
+            gagent_check_type = with_migrate
+            mig_speed = 512M
+            pre_migrate = mig_set_speed
```

Comment:
Useless since you use `self.vm.migrate()` directly instead of using sub_test `migration`

Reply:
I use it before migration.
self.vm.monitor.migrate_set_speed(params.get("mig_speed", "1G"))

Reply:
No, that is `mig_speed`, you can use it.  No usage for `pre_migrate`.

Reply:
`pre_migrate` is used to run some functions before migrate start, the logic is in script `migration.py`. Nothing related with this case.

Reply:
Read migration.py/cfg, you are right.
Updated. pls have a check. thanks.